chapters/hi/_toctree.yml

```diff
@@ -0,0 +1,173 @@
+- title: 0. स्थापना
+  sections:
+  - local: chapter0/1
+    title: परिचय
+
+- title: 1. ट्रांसफार्मर मॉडल

```

Comment:
Thank you for translating the whole table of contents, but we should only include the chapters / sections that have been translated (I need to make this clearer in the instructions, sorry!)

This means you can remove everything from here downwards